File: 프로그래머스/0/120843. 공 던지기/공 던지기.java
class Solution {
    public int solution(int[] numbers, int k) {
        int answer = 0;
        int index = 0;
        for(int i=0; i<k; i++) {
            if(index+2<numbers.length) {
                answer = numbers[index];
                index += 2;
            } else if(index+2 == numbers.length){
                answer = numbers[index];
                index -= index;
            } else {
                answer = numbers[index];
                index -= index-1;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120843. 공 던지기/공 던지기.js
function solution(numbers, k) {
    var answer = 0;
    let index = 0;
    for(let i=0; i<k; i++) {
        if(index+2 >= numbers.length) {
            answer = numbers[index];
            index += 2;
            index -= numbers.length;
        } else {
            answer = numbers[index];
            index += 2;
        }
    }
    return answer;
}